src/dmd/aggregate.d

```diff
@@ -219,9 +219,8 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
             return 0;
         }
 
-        for (size_t i = 0; i < members.dim; i++)
```

Comment:
[This loop can add members, you cant convert it](https://github.com/dlang/dmd/pull/8544#discussion_r208787525).
That's why there was _little resistance_, as you said,